anta/tests/snmp.py

```diff
@@ -114,3 +114,63 @@ def test(self) -> None:
             self.result.is_failure(f"SNMP IPv6 ACL(s) not configured or active in vrf {self.inputs.vrf}: {not_configured_acl_list}")
         else:
             self.result.is_success()
+
+
+class VerifySnmpLocation(AntaTest):
+    """
+    This class verifies the SNMP location of a device.
+
+    Expected results:
+        * success: The test will pass if the SNMP location matches the provided input.
+        * failure: The test will fail if the SNMP location does not match the provided input.
+    """
+
+    name = "VerifySnmpLocation"
+    description = "Verifies the SNMP location of a device."
+    categories = ["security"]
```

Comment:
going to change the categories in next commit. missed to update. Same for  VerifySnmpContact

Reply:
Yes please. There is an `snmp` category :)
```suggestion
    categories = ["snmp"]
```